src/main/java/seedu/address/logic/commands/room/InitRoomCommand.java

```diff
@@ -39,8 +41,10 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_ZERO_CANNOT_BE_AN_INPUT);
         } else if (numOfRooms < 0) {
             throw new CommandException(MESSAGE_NEGATIVE_VALUES_CANNOT_BE_INPUT);
+        } else if (numOfRooms > 5000) {
```

Comment:
Would be good to define `5000` as a constant `MAX_NUM_OF_ROOM`!

Reply:
Okay thanks.